server/controllers/index.js

```diff
@@ -1,52 +1,90 @@
 const lodash = require('lodash');
+const { Product } = require('../../database/models/product.js');
 
-const { Product } = require('../../database/models/product.js', {
-  useNewUrlParser: true,
-  useUnifiedTopology: true,
-});
-
-const title = (req, res) => {
-  const id = req.params.product_id.split(':').join('');
-  Product.find({ product_id: id })
-    .then((result) => {
-      // eslint-disable-next-line no-underscore-dangle
-      const productObj = result[0]._doc;
-      // testing purposes: console.log(productObj);
-      const productInfo = lodash.omit(productObj, ['_id', '__v']);
-      res.status(200).send(productInfo);
-    })
-    .catch((err) => {
-      console.log(err);
-      res.status(500).send({ error: 'Something Broke!' });
-    });
+const getProduct = async (req, res) => {
+  const id = req.params.product_id;
+
+  try {
+    const productInfo = await Product.find({product_id: id});
+    const productDetails = lodash.omit(productInfo[0]._doc, ['_id', '__v']);
```

Comment:
I haven't seen lodash.omit used before, I read its docs and this looks like a great spot to use it.